gulpfile.js

```diff
@@ -8,7 +8,7 @@ var concat = require( 'gulp-concat' );
 var nano = require('gulp-cssnano' );
 var handlebars = require( 'gulp-compile-handlebars' );
 var rename = require( 'gulp-rename' );
-var files = [ 'index.html', '../assets/css/style.css' ];
+var files = [ 'index.html', './assets/css/styles.css' ];
```

Comment:
This was not working already, right?

Reply:
no haha.. now with the serial build, it failed to find this file in the build time.